components/modelregistry/modelregistry.go

```diff
@@ -118,6 +114,10 @@ func (m *ModelRegistry) ReconcileComponent(ctx context.Context, cli client.Clien
 		}
 
 		// Create model registries namespace
+		// we should not apply this label to the namespace, as it triggered namspace deletion during operator uninstall
+		// modelRegistryLabels = cluster.WithLabels(
```

Comment:
I'm curious, any specific reason to keep this commented code ?

Reply:
for maintainance reason only